imports/plugins/included/jobcontrol/server/jobs/cleanup.js

```diff
@@ -0,0 +1,47 @@
+import { Jobs } from "/lib/collections";
+import { Hooks, Logger } from "/server/api";
+
+if (Hooks) {
+  Hooks.Events.add("afterCoreInit", () => {
```

Comment:
Should me maybe create an `onJobsServerStarted` hook for scheduled jobs that kick off at startup?  I don't think the order actually matters much in this case, but it feels like it would make sense since there's a callback for `Jobs.startJobServer()`.